samples/scala-protobuf-tracing/project/plugins.sbt

```
@@ -0,0 +1,4 @@
+addSbtPlugin("io.kalix" % "sbt-kalix" % "1.4.1")
```

```suggestion
addSbtPlugin("io.kalix" % "sbt-kalix" % System.getProperty("kalix-sdk.version", "1.4.1"))
```
Like in the other samples.